flang/test/Runtime/parallel_do_if_false.f90

```diff
@@ -0,0 +1,27 @@
+!RUN: %flang %s -o %t -fopenmp  && %t | FileCheck %s
```

Comment:
I don't think this test can work here. It needs to be moved into `openmp'/runtime/test`, but it currently lacks of the infrastructure to handle it in `libomp`. Check `openmp/libomptarget/test/lit.site.cfg.in` on how we handle Fortran test cases in `libomptarget`.

Reply:
Thank you for suggestion, I'll look into it.

Reply:
Hi @shiltian I've added the test, could you please review.